packages/nuxt3/src/app/composables/nuxt-link.ts

```diff
@@ -0,0 +1,242 @@
+    }
+  }
+
+  let RouterLink: ConcreteComponent
+  const getRouterLink = (): ConcreteComponent => {
+    if (!RouterLink) {
+      RouterLink = resolveComponent('RouterLink') as ConcreteComponent
+    }
+    return RouterLink
+  }
+
+  return defineComponent({
+    name: options.componentName,
+    props: {
+      // Routing
+      to: {
+        type: [String, Object] as PropType<string | RouteLocationRaw>,
+        default: undefined,
+        required: false
+      },
+      href: {
+        type: [String, Object] as PropType<string | RouteLocationRaw>,
+        default: undefined,
+        required: false
+      },
+
+      // Attributes
+      target: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+      rel: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+      noRel: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      },
+
+      // Prefetching
+      prefetch: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      },
+      noPrefetch: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      },
+
+      // Styling
+      activeClass: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+      exactActiveClass: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+      prefetchedClass: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+
+      // Vue Router's `<RouterLink>` additional props
+      replace: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      },
+      ariaCurrentValue: {
+        type: String as PropType<string>,
+        default: undefined,
+        required: false
+      },
+
+      // Edge cases handling
+      external: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      },
+
+      // Slot API
+      custom: {
+        type: Boolean as PropType<boolean>,
+        default: undefined,
+        required: false
+      }
+    },
+    setup (props, { slots }) {
+      const router = useRouter() as Router | undefined
+
+      // Resolving `to` value from `to` and `href` props
+      const to = computed<string | RouteLocationRaw>(() => {
+        checkPropConflicts(props, 'to', 'href')
+
+        return (props.to ?? props.href) || '' // Defaults to empty string (won't render any `href` attribute)
+      })
+
+      // Resolving link type
+      const isExternal = computed<boolean>(() => {
+        if (props.external !== undefined) {
+          // Else if `external` prop is used
+          return props.external
+        } else if (props.target) {
```

Comment:
_self needs to be excluded: 

```suggestion
        } else if (props.target && props.target !== '_self') {
```

Reply:
Good catch!